test/integration/serverless/src/java-tests/log/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Mon Nov 29 10:50:44 EST 2021
+groupId=com.integration-test
+artifactId=log
+version=dev
```

Comment:
I am not familiar with Maven or the Java ecosystem in general, but do we need to commit all of these files? Generally files that are generated by a build do not need to be committed in the repository because they will be regenerated when someone runs the tests.

Reply:
no this should have been in git ignore